.npmrc

```diff
@@ -0,0 +1 @@
+registry=http://nexus.containerstore.com/nexus/repository/npm-group/
```

Comment:
@andyasberry I don't think we should add this, limits the usability of this test

Reply:
I need to remove this and will today.

Reply:
Done 👍